resqpy/surface/_surface.py

```diff
@@ -636,7 +636,7 @@ def set_from_point_set(self,
             assert -90.0 < saucer_parameter < 90.0, f'simple saucer angle parameter must be less than 90 degrees; too big: {saucer_parameter}'
```

Comment:
Let's strip out the old option, so remove the (saucer_parameter > 1.0 or saucer_parameter < 0.0) please. Also, please update the docstring notes to strip out the old option (1)